services/net/contentmigration/TNO.Services.ContentMigration.csproj

```diff
@@ -11,14 +11,15 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\..\libs\net\services\TNO.Services.csproj" />
+    <!-- <ProjectReference Include="..\..\..\libs\net\services\TNO.Services.csproj" /> -->
   </ItemGroup>
 
   <ItemGroup>
     <PackageReference Include="LinqKit" Version="1.3.7" />
     <PackageReference Include="System.CommandLine.DragonFruit" Version="0.3.0-alpha.20070.2" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="9.0.0" />
-    <PackageReference Include="Aspire.Oracle.EntityFrameworkCore" Version="9.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="8.0.1" />
+    <PackageReference Include="Aspire.Oracle.EntityFrameworkCore" Version="8.2.2" />
+    <PackageReference Include="TNO.Services" Version="1.0.0" />
```

Comment:
did you mean version 2.0.0? the above TNO.Services.csproj has update the version to 2.0.0.

Reply:
No.  This points to version 1.0.0 because that version is .NET8.0.  Version 2.0.0 would be .NET9.0 if we built new packages.